source/js/components/petition/donation-modal.jsx

```diff
@@ -74,18 +85,21 @@ class DonationModal extends Component {
       this.donateURL = `?form=donate`;
     }
     return (
-      <div className="modal-content" role="dialog">
+      <div
+        className={classNames(`modal-content`, MODAL_CONTENT_CLASS)}
+        role="dialog"
+      >
         <button
           ref={(e) => (this.closeButton = e)}
-          className="close"
+          className={CLOSE_BUTTON_CLASS}
           data-dismiss="modal"
           aria-label="Close"
           tabIndex="0"
         >
           <span aria-hidden="true">&times;</span>
         </button>
 
-        <div className="modal-body">
+        <div className={classNames(`modal-body`, MODAL_BODY_CLASS)}>
```

Comment:
Similarly, we need to keep `modal-body` class here as we are still relying on Bootstrap's stylings for `.modal-body`